# simpro发送给AD的数据
header                         head
lane_info[]                    laneInfos
sensor_info[]                  sensorStates
sensor_detection_info[]        sensorObjects
contact_point[]                contactPoints
traffic_sign[]                 trafficSigns
road_state[]                   roadStates
environment[]                  environments
traffic_light[]                trafficLights
road_query[]                   roadQueries
mil_ego_state[]                egoStates
mil_object_state[]             milObjStates
mil_roadmark[]                 milRoadMarks
init_param[]                   initParams
mil_amesim_signal[]            milAmesimSignals
activate_controller[]          activateControllers
assign_controller[]            assignControllers
override_controller[]          overrideControllers
custom_command[]               customCommands
parkingSpace[]                 parkingSpaces
driver_ctrl[]                  driverCtrls
sync[]                         syncs   
imu_data[]                     imuDatas     
gnss_gga[]                     gnssGgaS
uwb_data[]                     uwbDatas
model_outline[]                modelOutlines
rsu_vnfp_info                  rsuVnfpInfo